source-control/versioning/readme.md

```diff
@@ -30,9 +30,9 @@ Updating the major / minor version indicates changes of the API / interfaces / c
 
 ## Semantic Versioning
 
-Semantic Versioning is a concept of calculating the version number automatically based on a certain source code respository.
+Semantic Versioning is a concept of calculating the version number automatically based on a certain source code rspository.
```

Comment:
repository :)

Reply:
How embarrassing for me.  Working on fixing - thank you